### ISSUE TITLE
rate parsing shouldn't use powers of 2

### ISSUE THREAD

**Issue by edwintorok** [frenetic-lang/ocaml-topology#59]
According to https://en.wikipedia.org/wiki/Binary_prefix#Data_transmission_and_clock_rates and
https://en.wikipedia.org/wiki/Gigabit_Ethernet  a GbE link can transfer 1,000,000,000 bits per second,
and a 100 Mbit/s link can transfer 100,000,000 bits per second.

But you parse `1Gbps` as `2^30` which doesn't seem right:
https://github.com/frenetic-lang/ocaml-topology/blob/master/lib/Network_Common.ml#L20

**Comment by jnfoster** [comment]
Yup. Note that most development now happens in the main [frenetic](https://github.com/frenetic-lang/frenetic) repository
